.github/workflows/release.yml

```
@@ -20,8 +20,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.22.7
-          check-latest: true
+          go-version: 1.22.0
```

Shouldn't we use Go v1.23 here too?
```suggestion
          go-version: 1.23.0
```